### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by mrleewong**

### CODE REVIEW

File: _data/wallets-collection.yml [hunk 1]
```diff
@@ -42,13 +42,13 @@
...
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-win64.zip"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3.1/dashcore-0.12.3.1-win64.zip"
     -
       label: Hash File
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/SHA256SUMS.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3.1/SHA256SUMS.asc"
     -
       label: PGP
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-win64.zip.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3/dashcore-0.12.3-win64.zip.asc"
```
Comment (UdjinM6) [inline _data/wallets-collection.yml]: There is an inconsistency - all other `PGP`s are disabled. Also, there is no such file.

File: _data/wallets-collection.yml [hunk 2]
```diff
@@ -122,13 +122,13 @@
...
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-linux32.tar.gz"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3.1/dashcore-0.12.3.1-i686-pc-linux-gnu.tar.gz"
     -
       label: Hash File
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/SHA256SUMS.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3/SHA256SUMS.asc"
     -
       label: PGP
-      url: "https://github.com/dashpay/dash/releases/download/v0.12.2.3/dashcore-0.12.2.3-linux32.tar.gz.asc"
+      url: "https://github.com/dashpay/dash/releases/download/v0.12.3/dashcore-0.12.3-linux32.tar.gz.asc"
```
Comment (UdjinM6) [inline _data/wallets-collection.yml]: Same

**Comment by mrleewong** [comment]
@UdjinM6 fixed

**Comment by UdjinM6** [comment]
Hmm... some versions in `de` and `ru` disappeared instead of being updated...
Also, `dashcore-0.12.3.1-arm-linux-gnueabihf.tar.gz` is what used to be RPi files, so maybe we could just update links instead of commenting the whole section out?

**Comment by mrleewong** [comment]
@UdjinM6 `de` and `ru` will take default key from translation (English) it will helps to take translation more consistent

**Comment by UdjinM6** [comment]
Ah, I see. Ok.

**Pull request merged**